bazel/external/quiche.BUILD

```diff
@@ -147,6 +151,78 @@ envoy_cc_library(
     deps = ["@envoy//source/extensions/quic_listeners/quiche/platform:spdy_platform_unsafe_arena_impl_lib"],
 )
 
+envoy_cc_library(
+    name = "spdy_core_spdy_alt_svc_wire_format_lib",
+    srcs = ["quiche/spdy/core/spdy_alt_svc_wire_format.cc"],
+    hdrs = ["quiche/spdy/core/spdy_alt_svc_wire_format.h"],
+    copts = quiche_copt,
+    repository = "@envoy",
+    visibility = ["//visibility:public"],
+    deps = [
+        ":spdy_platform",
+    ],
+)
+
+envoy_cc_library(
+    name = "spdy_core_spdy_header_block_lib",
+    srcs = [
```

Comment:
nit: fold "srcs" and "hdrs" into a single line.

Reply:
done